app/controllers/merchant/items_controller.rb

```diff
@@ -43,16 +43,27 @@ def destroy
     redirect_to '/merchant/items'
   end
 
+  def edit
+    @item = Item.find(params[:id])
+  end
+
+  def edit_update
```

Comment:
Is there a reason this is edit_update instead of update?

Reply:
OK, I see! Maybe later we can change this to update, and the other "update" to `update_status` or something.